arch/arm64/boot/dts/qcom/msm8953-lenovo-kuntao.dts

```diff
@@ -63,11 +106,69 @@
 	status = "okay";
 };
 
+&lpass {
+	status = "okay";
+};
+
+&mdss {
+	status = "okay";
+};
+
+&mdss_dsi0 {
+	vdda-supply = <&pm8953_s3>;
+	vddio-supply = <&pm8953_l6>;
+
+	panel: panel@0 {
+		compatible = "lenovo,kuntao-549";
+		reg = <0>;
+
+		pinctrl-0 = <&pmx_mdss_default>;
+		pinctrl-1 = <&pmx_mdss_sleep>;
+		pinctrl-names = "default", "sleep";
+
+		reset-gpios = <&tlmm 61 GPIO_ACTIVE_LOW>;
+
+		vsp-supply = <&lab_vreg>;
+		vsn-supply = <&ibb_vreg>;
+
+		port {
+			panel_in: endpoint {
+				remote-endpoint = <&mdss_dsi0_out>;
+			};
+		};
+	};
+
+	status = "okay";
```

Comment:
@barni2000 it is not compiling the dts when `status = "okay"` is in the last

Reply:
properties need to come before any subnode, but `status` should be the last property before the first subnode.